### PR TITLE
POC Support nullable parameters

### PR DESCRIPTION
**Pull request by funivan**
#### What?

A description about what this pull request implements and its purpose. Try to be detailed and describe any technical
details to simplify the job of the reviewer.

#### Tickets / Documentation

Add links to any relevant issues and documentation.

- [Link 1](http://example.com)
- ...

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -21,7 +21,7 @@
   "minimum-stability": "dev",
   "prefer-stable": true,
   "require": {
-    "php": "^7.3 || ^8.0",
+    "php": "^7.4 || ^8.0",
```
Comment (icatalina) [inline composer.json:24]: Is this necessary?

**Review by icatalina — COMMENTED** [review]
👏 Thanks man! would have taking me days 😅 🙇

File: composer.json (hunk 1 above)
Comment (funivan) [reply composer.json:24]: 1. Typed property appeared in php 7.4
2. Do we still support 7.3 in BC?

Comment (icatalina) [reply composer.json:24]: 1. We are not dependent on type properties, are we? type arguments iirc, and nullable ones are supported from 7.1
2. No, but this repo is public, if not necessary I see no reason to upgrade the minimum required version, wdyt?

Comment (funivan) [reply composer.json:24]: 1. https://github.com/bigcommerce/injector/pull/16/files#diff-9bf09a09806ea01fdfb744f558f7006011d5a81fb37806e82d4509d007596cf6R11 we can change it.
2. https://www.php.net/supported-versions.php I recommend supporting at least a stable version of the PHP.

**Pull request closed without merging**